forms-flow-submissions/src/Routes/SubmissionListing.tsx

```diff
@@ -173,7 +179,15 @@ const TaskSubmissionList: React.FC = () => {
    <div className="main-layout-container">
       {/* Left Panel - Collapsible Search Form */}
       <div className="left-panel">
-        <CollapsibleSearch />
+        <CollapsibleSearch
+          isOpen={true}
+          hasActiveFilters={false}
+          inactiveLabel="No Filters"
+          activeLabel="Filters Active"
+          onToggle={() => { }}
```

Comment:
The handleToggle() function is to be added int the future isn't it? or is it having some other logics like do nothing on toggle? 
Or is it having any other dependency ticketes?